internal/pkg/service/buffer/dependencies/dependencies.go

```diff
@@ -69,7 +69,7 @@ func NewServiceDeps(
 	// Create public dependencies - load API index
 	startTime := time.Now()
 	logger.Info("loading Storage API index")
-	publicDeps, err := dependencies.NewPublicDeps(ctx, baseDeps, storageAPIHost)
+	publicDeps, err := dependencies.NewPublicDeps(ctx, baseDeps, storageAPIHost, false)
```

Comment:
Mohlo by byt z toho prosim `WithComponents(false)`? 
Ono sa to lepsie cita a vies co to robi, ... a nechcem tam pridavat asi dalsie `bool flags`.

Reply:
aha, ok

Reply:
example https://github.com/keboola/keboola-as-code/blob/main/internal/pkg/service/buffer/worker/distribution/config.go